### DISCUSSION TITLE
Does hasura action supports application/vnd.api+json content-type?

### DISCUSSION THREAD

**Discussion by freeRajeev** [hasura/graphql-engine#10133] [Question]
I am trying to call a rest API which accepts Content-type: application/vnd.api+json and Accept: application/vnd.api+json.
I have added these two variable in header but looks like Hasura is overriding these header. Also there in request transformation section there is not option to select application/vnd.api+json so I have selected application/json and provided request transformation logic.

**Comment by fg24davies** [comment]
Not an answer - same issue but with a different content type:
I am trying to do a POST to an api that accepts `Content-Type: Application/ld+json` but I think Hasura is overriding it to json only.
Are `application/json` and `application/x-www-form-urlenconded` the only content types supported?
